Gemfile.base

```diff
@@ -264,6 +264,9 @@ group :development, :test do
 
   # for `rake doc:liquid:generate` and similar
   gem 'source2swagger', git: 'https://github.com/3scale/source2swagger'
+
+  # for creating model and controller UML diagrams
+  gem 'railroady'
```

Comment:
Old gem. Long time without update. List of issues completely abandoned apparently.

I used to work with https://github.com/voormedia/rails-erd in the past, but possibly worse nowadays.

Reply:
Yeah was just a 10 minute exercise to create uml for qe. Seems to work. Last commit is from 2018 so not completely abandoned it seems. No need to merge this.